packages/now-next/src/index.ts

```diff
@@ -400,6 +404,17 @@ export const build = async ({
       routes: [
         // TODO: low priority: handle trailingSlash
 
+        // Add top level rewrite for basePath if provided
+        ...(nextBasePath
+          ? [
+              {
+                src: `${nextBasePath}/(.*)`,
```

Comment:
Does this work for someone visiting just `<basePath>` (think about no trailing slash)? Should this be the following?
```suggestion
                src: `^${nextBasePath}(?:$|/(.*))$`,
```

Reply:
☝️ 

![image](https://user-images.githubusercontent.com/616428/71688680-51306500-2d6e-11ea-9eac-8dfee52bbdf5.png)

Reply:
Updated